### ISSUE TITLE
down/ and up/ directories

### ISSUE THREAD

**Issue by ghost** [RowleyGroup/ACFCalculator#1]
Hey!

I divided my membrane into 60 bins and after I run umbrella sampling simulations using Gromacs, I have 60 different umbrella{i}_pullf.xvg and umbrella{i}_pullx.xvg (i= 0 to 50) files.  Do I need to put these files in two down/ and up/ directories and then run the setup.sh and getDall.py to get the diffusion curve?
Actually I did and the result(attached photo) was nonsense. 

I would be appreciated if you can advise me to fix it.

<img width="934" alt="Screenshot 2020-09-15 at 6 15 12 PM" src="https://user-images.githubusercontent.com/42797415/93198308-6d9af300-f77f-11ea-92c4-952ffeecf3c3.png">

**Comment by cnrowley** [comment]
How frequently are you saving your steps from your simulation? For
diffusion, you usually have to save them at every step.

**Comment by ghost** [comment]
Thanks for your reply.

I just set -s value to 2 as it was in the original script. I have run every bin of membrane for 50 ns so it means I should set the value of s to 25000000?
Also, do you know what's the optimum value for --maxcorr ? it looks changing its value, doesn't change the curve for my simulation

**Comment by cnrowley** [comment]
No, how often are you saving the position of the solute in your MD
simulation?

**Comment by ghost** [comment]
I have set nstxout-compressed  = 500 in my simulation which is 50ns run time for every bin (60 bins in total).
Can you advise me on how I should set the input values to get the right diffusion curve?

Based on my understanding, I should set -f value to 1 as I used Gromacs, -s value to 2000 as every step in my simulation is 2 ps which is 2000 fs and --maxcorr to 50000 , but I face below error for every bin:

#Time step of 2000 fs will be used.
#No cutoff will be used to integrate the ACF.
#D(s) will be saved in out_up_sim_2.out
#var 1.15969e+10
#varVel 6.54981e-30
#WARNING: ACF has only decayed to 94.4956% of initial value at the end of the region of integration. 
libc++abi.dylib: terminating with uncaught exception of type boost::wrapexcept<std::domain_error>: Error in function boost::math::tools::toms748_solve<double>: Parameters a and b do not bracket the root: a=9.9999999999999995e-07
setup.sh: line 6: 34857 Abort trap: 6           /Users/javad/Downloads/ACFCalculator-master/build/ACFcalculator -i umbrella${i}_pullx.xvg -t gromacs -a acf_up_sim_${i}.dat -o out_up_sim_${i}.out -f 1 -s 2000 --maxcorr 50000

**Comment by cnrowley** [comment]
This is a somewhat different issue. You have to save a time series for the
coordinate you are calculating the diffusion coefficient for at every step.
I would recommend reading the paper
https://pubs.acs.org/doi/abs/10.1021/acs.jctc.6b00747 carefully before
attempting to do these calculations.
